src/web/models.py

```diff
@@ -14,6 +13,11 @@ class TechGroup(HandyHelperBaseModel):
     enabled = models.BooleanField(default=True)
     # platform = models.ForeignKey("EventPlatform", blank=True, null=True, on_delete=models.SET_NULL)
     homepage = models.URLField(blank=True, null=True)
+    icon = models.CharField(
```

Comment:
do we want this to be nullable?

Reply:
No, https://docs.djangoproject.com/en/5.0/ref/models/fields/

> Avoid using [null](https://docs.djangoproject.com/en/5.0/ref/models/fields/#django.db.models.Field.null) on string-based fields such as [CharField](https://docs.djangoproject.com/en/5.0/ref/models/fields/#django.db.models.CharField) and [TextField](https://docs.djangoproject.com/en/5.0/ref/models/fields/#django.db.models.TextField). If a string-based field has null=True, that means it has two possible values for “no data”: NULL, and the empty string.